epub34/authoring/index.html

```diff
@@ -5795,27 +5807,38 @@ <h4>NCX</h4>
 			<h2>EPUB content documents</h2>
 
 			<section id="sec-xhtml">
-				<h3>XHTML content documents</h3>
+				<h3>HTML content documents</h3>
+
+				<div class="ednote">
+					<p>Support for the HTML syntax is still an open issue in the EPUB 3.4 revision. This specification
+						introduces it as a core media type and also minimally adds requirements for its authoring in
+						this section to begin making readers aware of the proposed change.</p>
+
+					<p>Much of this specification, and the Reading Systems specification [[epub-rs-34]], continues to
+						refer only to XHTML content documents and their authoring requirements, a number of which need
+						resolving (e.g., providing an alternative to the <code>epub:type</code> attribute). These
```

Comment:
As I commented elsewhere, I would prefer to document the `epub:type` alternative right away, as it is probably the most fundamental question that needs resolution, and put it out there early is the best. I am not sure what other example we would put here (maybe whether we allow the nav file to be HTML?)